scripts/install_prerequisites.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
+
+
+sudo apt-get update
+ 
+sudo apt-get install git 
+
+sudo apt-get install unzip
+ sudo apt-get install default-jre
```

Comment:
Is this needed? It might not be anymore, actually, in which case it'd be nice to remove it.